src/layout/selector.ts

```diff
@@ -15,7 +16,7 @@ export default function(tag: string, prefix: string, attributes: Attributes): st
         default:
             if (prefix === null) { return empty; }
             tag = tag.indexOf(Constant.SVG_PREFIX) === 0 ? tag.substr(Constant.SVG_PREFIX.length) : tag;
-            let s = "id" in attributes && attributes["id"].length > 0 ? `${tag}#${attributes.id}` : `${prefix}${tag}`;
+            let s = "id" in attributes && attributes["id"].length > 0 ? `${tag}#${attributes.id}` : `${prefix}${tag}${suffix}`;
```

Comment:
>${suffix} [](start = 116, length = 9)

Add a delimiter here between the tag and suffix.

Reply:
Similar to prefix, suffix contains the delimiter. If suffix is empty, we don't want a delimiter. This is to ensure that we end up with valid selectors in the end.